pom.xml

```diff
@@ -46,8 +46,7 @@
     </scm>
 
     <properties>
-        <jenkins.version>1.609.3</jenkins.version>
-        <workflow.version>1.14</workflow.version>
+        <jenkins.version>1.642.3</jenkins.version>
```

Comment:
Why are we bumping to 1.642.3 jenkins dependency?  This removes usability for many customers.